pkg/k8s/watchers/subscriber/ceb.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscriber
+
+import (
+	cilium_v2a1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+)
+
+// NewCEB creates a new ceb subscriber list.
+func NewCEB() *CebList {
```

Comment:
```suggestion
func NewCEB() *CEBList {
```

Please `s/Ceb/CEB/g` across this file

Reply:
Done.